lib/spack/spack/new_installer.py

```diff
@@ -695,10 +706,10 @@ def next(self, direction: int = 1) -> None:
         self.tracked_build_id = new_build_id
 
         # Tell the user we're following new logs, and instruct the child to start sending them.
-        print(
-            f"\n==> Following logs of {new_build.name}" f"\033[0;36m@{new_build.version}\033[0m",
-            flush=True,
```

Comment:
Any specific reason to change the `print` call into 2 calls, instead of using `file=self.stdout`?